### PR TITLE
add name to authors

### LINKED ISSUE #1: Add names to author file

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #1

### CODE REVIEW

File: authors.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-1.
+1. Micah Godbolt
 2.
```
Comment (micahgodbolt) [inline authors.md]: Nice change

Comment (micahgodbolt) [reply authors.md]: Change this to a const instead of let

**Review by WillHab — APPROVED** [review]
Looks good to me!

**Comment by micahgodbolt** [comment]
:shipit:

**Pull request merged**